Index: src/main/java/org/example/Piece.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/Piece.java b/src/main/java/org/example/Piece.java
--- a/src/main/java/org/example/Piece.java	
+++ b/src/main/java/org/example/Piece.java	
@@ -24,43 +24,43 @@
         {new Tile(new Color(175, 175, 225))}
     };
 
-private static final Tile[][] STRAIGHT2x_PIECE = new Tile[][]{
+    private static final Tile[][] STRAIGHT2x_PIECE = new Tile[][]{
         {new Tile(Color.yellow),new Tile(Color.yellow)}
-        };
+    };
 
-private static final Tile[][] STRAIGHT3x_PIECE = new Tile[][]{
+    private static final Tile[][] STRAIGHT3x_PIECE = new Tile[][]{
         {new Tile(Color.orange),new Tile(Color.orange),new Tile(Color.orange)}
-        };
+    };
 
-private static final Tile[][] STRAIGHT4x_PIECE = new Tile[][]{
+    private static final Tile[][] STRAIGHT4x_PIECE = new Tile[][]{
         {new Tile(Color.magenta),new Tile(Color.magenta),new Tile(Color.magenta),new Tile(Color.magenta)}
-        };
+    };
 
-private static final Tile[][] STRAIGHT5x_PIECE = new Tile[][]{
+    private static final Tile[][] STRAIGHT5x_PIECE = new Tile[][]{
         {new Tile(Color.red),new Tile(Color.red),new Tile(Color.red),new Tile(Color.red),new Tile(Color.red)}
-        };
+    };
 
-private static final Tile[][] ELLE2x_PIECE = new Tile[][]{
+    private static final Tile[][] ELLE2x_PIECE = new Tile[][]{
         {new Tile(Color.green), new Tile()},
         {new Tile(Color.green), new Tile(Color.green)}
-        };
+    };
 
-private static final Tile[][] ELLE3x_PIECE = new Tile[][]{
+    private static final Tile[][] ELLE3x_PIECE = new Tile[][]{
         {new Tile(Color.darkGray),new Tile(),new Tile()},
         {new Tile(Color.darkGray),new Tile(),new Tile()},
         {new Tile(Color.darkGray),new Tile(Color.darkGray),new Tile(Color.darkGray)}
-        };
+    };
 
-private static final Tile[][] REVERSEDELLE2X_PIECE = new Tile[][]{
+    private static final Tile[][] REVERSEDELLE2X_PIECE = new Tile[][]{
         {new Tile(), new Tile(Color.green)},
         {new Tile(Color.green), new Tile(Color.green)}
-        };
+    };
 
-private static final Tile[][] REVERSEDELLE3X_PIECE = new Tile[][]{
+    private static final Tile[][] REVERSEDELLE3X_PIECE = new Tile[][]{
         {new Tile(), new Tile(), new Tile(Color.darkGray)},
         {new Tile(), new Tile(), new Tile(Color.darkGray)},
         {new Tile(Color.darkGray), new Tile(Color.darkGray), new Tile(Color.darkGray)}
-        };
+    };
     private enum ShapeType{
         SQUARE2x,
         SQUARE3x,
